test/test_format_string.cpp

```diff
@@ -17,9 +17,12 @@
 #include <string>
 
 #include "./allocator_testing_utils.h"
+
 #include "rcutils/allocator.h"
 #include "rcutils/format_string.h"
 
+#include "./mocking_utils.hpp"
```

Comment:
Would it make sense to group the two relative imports?

Reply:
It would.